engines/sherlock/scalpel/scalpel.cpp

```diff
@@ -0,0 +1,775 @@
+
+		// Write the text appropriate for the next frame
+		if (idx < 400)
+			_screen->transBlitFrom(creditsImages[0], Common::Point(10, 200 - idx), false, 0);
+		if (idx > 200)
+			_screen->transBlitFrom(creditsImages[1], Common::Point(10, 400 - idx), false, 0);
+
+		// Don't show credit text on the top and bottom ten rows of the screen
+		_screen->blitFrom(_screen->_backBuffer1, Common::Point(0, 0), Common::Rect(0, 0, SHERLOCK_SCREEN_WIDTH, 10));
+		_screen->blitFrom(_screen->_backBuffer1, Common::Point(0, SHERLOCK_SCREEN_HEIGHT - 10),
+			Common::Rect(0, SHERLOCK_SCREEN_HEIGHT - 10, SHERLOCK_SCREEN_WIDTH, SHERLOCK_SCREEN_HEIGHT));
+
+		_events->delay(100);
+	}
+
+	return true;
+}
+
+bool ScalpelEngine::showOfficeCutscene() {
+	_sound->playMusic("PROLOG4.MUS");
+	_animation->_gfxLibraryFilename = "TITLE2.LIB";
+	_animation->_soundLibraryFilename = "TITLE.SND";
+
+	bool finished = _animation->play("COFF1", 1, 3, true, 3);
+	if (finished)
+		finished = _animation->play("COFF2", 1, 0, false, 3);
+	if (finished) {
+		showLBV("note.lbv");
+
+		if (_sound->_voices) {
+			finished = _sound->playSound("NOTE1", WAIT_KBD_OR_FINISH);
+			if (finished)
+				finished = _sound->playSound("NOTE2", WAIT_KBD_OR_FINISH);
+			if (finished)
+				finished = _sound->playSound("NOTE3", WAIT_KBD_OR_FINISH);
+			if (finished)
+				finished = _sound->playSound("NOTE4", WAIT_KBD_OR_FINISH);
+		} else
+			finished = _events->delay(19000);
+
+		_events->clearEvents();
+		finished = _events->delay(500);
+	}
+
+	if (finished)
+		finished = _animation->play("COFF3", 1, 0, true, 3);
+
+	if (finished)
+		finished = _animation->play("COFF4", 1, 0, false, 3);
+
+	if (finished)
+		finished = scrollCredits();
+
+	if (finished)
+		_screen->fadeToBlack(3);
+
+	_animation->_gfxLibraryFilename = "";
+	_animation->_soundLibraryFilename = "";
+	return finished;
+}
+
+void ScalpelEngine::loadInventory() {
+	Inventory &inv = *_inventory;
+
+	// Initial inventory
+	inv._holdings = 2;
+	inv.push_back(InventoryItem(0, "Message", "A message requesting help", "_ITEM03A"));
+	inv.push_back(InventoryItem(0, "Holmes Card", "A number of business cards", "_ITEM07A"));
+
+	// Hidden items
+	inv.push_back(InventoryItem(95, "Tickets", "Opera Tickets", "_ITEM10A"));
+	inv.push_back(InventoryItem(138, "Cuff Link", "Cuff Link", "_ITEM04A"));
+	inv.push_back(InventoryItem(138, "Wire Hook", "Wire Hook", "_ITEM06A"));
+	inv.push_back(InventoryItem(150, "Note", "Note", "_ITEM13A"));
+	inv.push_back(InventoryItem(481, "Open Watch", "An open pocket watch", "_ITEM62A"));
+	inv.push_back(InventoryItem(481, "Paper", "A piece of paper with numbers on it", "_ITEM44A"));
+	inv.push_back(InventoryItem(532, "Letter", "A letter folded many times", "_ITEM68A"));
+	inv.push_back(InventoryItem(544, "Tarot", "Tarot Cards", "_ITEM71A"));
+	inv.push_back(InventoryItem(544, "Ornate Key", "An ornate key", "_ITEM70A"));
+	inv.push_back(InventoryItem(586, "Pawn ticket", "A pawn ticket", "_ITEM16A"));
```

Comment:
Out of curiosity, why are these items hidden? How are they separated from the visible ones?